### ISSUE TITLE
Any interest in implementing the extended isolation forest version

### ISSUE THREAD

**Issue by Marcus-Rosti** [linkedin/isolation-forest#58]
https://arxiv.org/abs/1811.02141

**Comment by jverbus** [comment]
It would be nice to add the extended isolation forest (EIF) variation of the algorithm to this library. However, I do not have plans to do this in the near term. 

If someone wants to contribute an implementation, I'd be happy to review a short tech spec doc (likely best to align on what changes will be made before starting coding) and subsequent PRs. To have it accepted, we'd need to make sure it meets the standard of quality of the existing library, including:

1. Follows the style and structure of the existing library. Reuses and/or extends existing code in an elegant way. 
2. Has comprehensive unit tests.
3. Has comprehensive validation on open datasets that match the results in the paper.
4. Is well documented.

It would also be nice to include ONNX export capability for the EIF (like we have for the IF in this library), but this could be a nice to have.

**Comment by jverbus** [comment]
I've been working on EIF capability and will commit it soon. Stay posted!
